plugin.video.jlss/changelog.txt

```diff
@@ -0,0 +1 @@
+v0.0.1 - Initial version
```

Comment:
Changelog is deprecated, you can set a `<news></news>` element within addon.xml. See: https://kodi.wiki/view/addon.xml#.3Cnews.3E